bolt/lib/linear_solver/tests/test_compute_moments.py

```diff
@@ -33,6 +33,17 @@
                      energy=[1, 1, 1]
                      )
 
+@af.broadcast
+def maxwell_boltzmann(rho, T, p1_b, p2_b, p3_b, p1, p2, p3):
+
+    f = rho * (1 / (2 * np.pi * T))**(3 / 2) * \
+             af.exp(-1 * (p1 - p1_b)**2 / (2 * T)) * \
+             af.exp(-1 * (p2 - p2_b)**2 / (2 * T)) * \
+             af.exp(-1 * (p3 - p3_b)**2 / (2 * T))
+
```

Comment:
change to:

```
    f = rho * (1 / (2 * np.pi * T))**(3 / 2) \
            * af.exp(-1 * (p1 - p1_b)**2 / (2 * T)) \
            * af.exp(-1 * (p2 - p2_b)**2 / (2 * T)) \
            * af.exp(-1 * (p3 - p3_b)**2 / (2 * T))
```

Basically move the multiply operators to infront and align them. Follow this in general for all mathematical operations.

Reply:
Okay. I'll do that.